docs/report/report.tex

```diff
@@ -52,15 +52,92 @@ \subsubsection{Stretch Goals}
 \section{Projektplanung}
 
 \subsection{Projekt Management}
-\colorbox{red}{TODO:}ticket pool in github, gemeinsame Absprache (im team und mit Kunden) was in den aktuellen Milestone kommen soll, alles zentral, 3x daily standup, gemeinsamer google calendar, gemeinsamer Discord Server (für spontaneres und informelleres gemeinsames Arbeiten), interne team evaluation und regelmäßiges feedback
+Am Anfang des Projekt entschied sich das Team in Abstimmung mit macio für eine agile Projektorganisation.
+Diese Richtung des Projektmanagements zeichnet sich vor allem durch fortlaufende Produktentwicklung, kontinuierliches Feedback, kooperatives Arbeiten und Reaktionsfähigkeit bei Anforderungsänderungen aus.
+Diese Entscheidung geschah aus mehreren Gründen.
+Zum einen haben agile Methoden kein festes Endprodukt als Ziel, wie es beispielsweise bei klassischer Softwareentwicklung (Wasserfallmodell, V-Model, etc) der Fall ist.
+Solch ein festes Ende würde der geforderten Veröffentlichung als Open-Source-Projekt nicht gerecht werden, da solche per Definition erweiterbar sind.
+Des weiteren haben agile Entwicklungsprozesse vergleichsweise kürzere Veröffentlichungszyklen.
+Dies ermöglichte dem Team ein funktionierendes Produkt innerhalb des recht knappen Zeitrahmens eines Semesters\footnote{Dieses Semester wahr aufgrund besonderer CoViD19-bedingten Auflagen einige Wochen kürzer als ein typisches.} zu erstellen.
+\\
+Entsprechend der Entscheidung zu agiler Entwicklung wurde zu Beginn des Projekt keine feste Planung des Projektverlaufs aufgestellt.
+Stattdessen entwickelte das Team eine Methodik um iterativ im Projekt zu arbeiten.
+Orientiert wurde sich hierbei an der Methode Scrum.
+Aus dieser wurden vor allem die Einteilung in Projektiterationen fester Länge (Sprints) und die zugehörigen wiederkehrenden Meetings übernommen.
+Die Rollen der Teammitglieder in Scrum wurde nicht adaptiert, da sich im Team für eine Gleichverteilung der Aufgaben entschieden wurde.
+
+\subsubsection{Vor den Sprints}
+Den Anfang eines jeden Sprints stellte das Sprint-Planing dar.
+Im diesem stellte das Team eine Vision für die Sprintiteration auf.
+Diese wurde gemeinsam mit dem Kunden abgesprochen und gegebenenfalls abgeändert.
+Im Anschluss wurde die Produktvision vom Team in technisch orientierte Arbeitspakete aufgeteilt.
+Dies geschah anhand des geschätzten zeitlichen Arbeitsaufwands der einzelnen Änderungen.
```

Comment:
Vielleicht hier noch ergänzen, dass wir nicht wie in Scrum mit direkten Story Points gearbeitet haben sondern eher nach Gefühl, wie lange es dauern wird

Reply:
story points sind nicht teil von scrum, sondern eine hilfreiche Methode, die zusätzlich eingesetzt werden kann. Siehe [Scrum Guide](https://scrumguides.org/scrum-guide.html)